index.d.ts

```diff
@@ -25,6 +25,8 @@ declare module 'plaid' {
     offset?: number;
   }
 
+  interface TransactionsAllRequestOptions extends ItemRequestOptions {}
```

Comment:
Sorry to be pedantic, but I think it would be more idiomatic / explicit to call this `GetAllTransactionsRequestOptions`, saying `TransactionAllRequestOptions` out loud just sounds wrong to me.

Reply:
No problem, I understand :)